tools/benchmark.py

```diff
@@ -391,6 +391,20 @@ def start_pipelinerun_thread(body, namespace):
     logging.debug(f"Created PipelineRun: {json.dumps(response)[:100]}...")
 
 
+def fetch_current_concurrency(value):
+    # Check if the value is a number
+    if value.isdigit():
+        return int(value)
+    # Check if the value is a valid file path
+    if os.path.isfile(value):
```

Comment:
```suggestion
    elif os.path.isfile(value):
```
and then add a else that would ba raising Exception (just to be on a safe side).

Reply:
Oh, wait, maybe this is not needed, sorry.